server/service/src/print/label.rs

```diff
@@ -109,11 +128,18 @@ pub fn print_prescription_label(
                 ^A0,20
                 ^FO,220
                 ^FB575,3,0,C
-                ^FD{patient_details}\&{details}\&^FS
+                ^FD{patient_details}\&{details}^FS
 
                 ^XZ
                 "#
```

Comment:
Suggested updated spacings (the values in here came from finding out that the maximum width of the label is ~600 px and giving the label a left and right margin of 18. I tested it on the real printer, result below):

```suggestion
r#"
^XA
^FX CI command parameters:
^FX - encoding (28 = UTF-8)
^CI28

^A0,25
^FO18,18
^FB582,3,0,C
^FD{item_details}^FS


^FX Line
^FO18,65
^GB582,2,2^FS

^A0,25
^FO28,75
^FB562,5,0
^FD{item_directions}\&{warning}^FS

^FX Line
^FO18,210
^GB582,2,2^FS

^A0,20
^FO18,220
^FB582,3,0,C
^FD{patient_details}\&{details}^FS

^XZ
"#
```